src/Kerbero.Domain/NukiActions/Models/NukiSmartLockStateDto.cs

```diff
@@ -1,13 +1,30 @@
+using System.Text.Json.Serialization;
+
 namespace Kerbero.Domain.NukiActions.Models;
 
 public class NukiSmartLockState
```

Comment:
```suggestion
public class NukiSmartLockStateDto
```
nome classe diverso dal file